src/components/Typography/src/typography.module.css

```diff
@@ -0,0 +1,53 @@
+.denhaag-heading-1 {
+  font-size: var(--denhaag-heading-1-font-size);
+  font-family: var(--denhaag-heading-1-font-family);
+  font-weight: var(--denhaag-heading-1-font-weight);
+  font-style: var(--denhaag-heading-1-font-style);
+  line-height: var(--denhaag-heading-1-line-height);
+  letter-spacing: var(--denhaag-heading-1-letter-spacing);
+}
+
+.denhaag-heading-2 {
+  font-size: var(--denhaag-heading-2-font-size);
+  font-family: var(--denhaag-heading-2-font-family);
+  font-weight: var(--denhaag-heading-2-font-weight);
+  font-style: var(--denhaag-heading-2-font-style);
+  line-height: var(--denhaag-heading-2-line-height);
+  letter-spacing: var(--denhaag-heading-2-letter-spacing);
+}
+
+.denhaag-heading-3 {
+  font-size: var(--denhaag-heading-3-font-size);
+  font-family: var(--denhaag-heading-3-font-family);
+  font-weight: var(--denhaag-heading-3-font-weight);
+  font-style: var(--denhaag-heading-3-font-style);
+  line-height: var(--denhaag-heading-3-line-height);
+  letter-spacing: var(--denhaag-heading-3-letter-spacing);
+}
+
+.denhaag-heading-4 {
+  font-size: var(--denhaag-heading-4-font-size);
+  font-family: var(--denhaag-heading-4-font-family);
+  font-weight: var(--denhaag-heading-4-font-weight);
+  font-style: var(--denhaag-heading-4-font-style);
+  line-height: var(--denhaag-heading-4-line-height);
+  letter-spacing: var(--denhaag-heading-4-letter-spacing);
+}
+
+.denhaag-heading-5 {
+  font-size: var(--denhaag-heading-5-font-size);
+  font-family: var(--denhaag-heading-5-font-family);
+  font-weight: var(--denhaag-heading-5-font-weight);
+  font-style: var(--denhaag-heading-5-font-style);
+  line-height: var(--denhaag-heading-5-line-height);
+  letter-spacing: var(--denhaag-heading-5-letter-spacing);
+}
+
+.denhaag-paragraph {
+  font-size: var(--denhaag-paragraph-font-size);
+  font-family: var(--denhaag-paragraph-font-family);
+  font-weight: var(--denhaag-paragraph-font-weight);
+  font-style: var(--denhaag-paragraph-font-style);
+  line-height: var(--denhaag-paragraph-line-height);
+  letter-spacing: var(--denhaag-paragraph-letter-spacing);
+}
```

Comment:
Since we work together towards an implementation of every component that works for most organisations, it would have been very interesting if you would have started by taking the CSS implementation from Utrecht for their Heading and Paragraph component, and worked from there to see if anything was missing.

Perhaps it is not too late to give that a try?

https://github.com/nl-design-system/utrecht/blob/main/components/heading/index.css

https://github.com/nl-design-system/utrecht/blob/main/components/paragraph/index.css